apps/builder/app/builder/features/style-panel/style-sections.tsx

```diff
@@ -143,4 +145,5 @@ export const sections: {
   borders: BordersSection,
   outline: OutlineSection,
   effects: EffectsSection,
+  boxShadow: BoxShadows,
```

Comment:
inconsistent naming, should be BoxShadowSection

Reply:
not BoxShadowsSection?